beaker.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |s|
   # Run time dependencies
   s.add_runtime_dependency 'minitest', '~> 5.4'
   s.add_runtime_dependency 'json', '~> 1.8'
+  s.add_runtime_dependency 'minitar', '~> 0.5.4'
```

Comment:
This requirement should be below the comment about the json dependency. Or above both of them.